assets/js/hocs/with-categories.js

```diff
@@ -29,7 +29,9 @@ const withCategories = createHigherOrderComponent( ( OriginalComponent ) => {
 		loadCategories() {
 			this.setState( { loading: true } );
 
-			getCategories()
+			getCategories( {
+				show_review_count: this.props.showReviewCount || false,
```

Comment:
I know we are very inconsistent about this, but I wonder if we should add a definition of `showReviewCount` in `propTypes` and set the default `false` value in `defaultProps`, as it's done in `withReviews`:
https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/master/assets/js/base/hocs/with-reviews.js#L16

Reply:
They’re applied inconsistently, I think we can do a separate PR to introduce them to `js/hocs/*` files

Reply:
Sounds good.